handlers/validator.go

```diff
@@ -492,15 +497,12 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 				break
 			}
 		}
+
+		validatorPageData.ShowMultipleWithdrawalCredentialsWarning = hasMultipleWithdrawalCredentials(validatorPageData.Deposits)
```

Comment:
This depends on `validatorPageData.Deposits` being set which happens in this very Go routine.